invisible_cities/core/core_functions_performance.py

```diff
@@ -1,32 +0,0 @@
-
```

Comment:
What's the rationale behind removing this file? Where was it used?

Reply:
Nowhere, that's why it is removed. I think it was some kind of script to compare performances, that didn't belong here, in my opinion.